strawberry/cli/commands/server.py

```diff
@@ -54,13 +54,13 @@ def server(
     try:
         import starlette  # noqa: F401
         import uvicorn
-    except ImportError:
+    except ImportError as e:
         rich.print(
             "[red]Error: The debug server requires additional packages, "
             "install them by running:\n"
             r"pip install 'strawberry-graphql\[debug-server]'"
         )
-        raise typer.Exit(1)
+        raise typer.Exit(1) from e
```

Comment:
I disagree about raising from e here, cluttering the error output. We are catching this error as expected and print a text.

Reply:
I'm not even sure it is printed anywhere

Reply:
Hrm, I can ignore the ruff rule for this line in specific